src/MyPlot/provider/MySQLProvider.php

```diff
@@ -48,9 +48,10 @@ public function __construct(MyPlot $plugin, int $cacheSize = 0, array $settings
 		$this->db = new \mysqli($settings['Host'], $settings['Username'], $settings['Password'], $settings['DatabaseName'], $settings['Port']);
 		if($this->db->connect_error)
 			throw new \RuntimeException("Failed to connect to the MySQL database: " . $this->db->connect_error);
-		$this->db->query("CREATE TABLE IF NOT EXISTS plots (id INT PRIMARY KEY AUTO_INCREMENT, level TEXT, X INT, Z INT, name TEXT, owner TEXT, helpers TEXT, denied TEXT, biome TEXT, pvp INT);");
+		$this->db->query("CREATE TABLE IF NOT EXISTS plots (id INT PRIMARY KEY AUTO_INCREMENT, level TEXT, X INT, Z INT, name TEXT, owner TEXT, helpers TEXT, denied TEXT, biome TEXT, pvp INT, price FLOAT);");
 		try{
 			$this->db->query("ALTER TABLE plots ADD COLUMN pvp INT AFTER biome;");
+			$this->db->query("ALTER TABLE plots ADD COLUMN price FLOAT AFTER pvp;");
```

Comment:
this needs its own try/catch in case the line above crashes

Reply:
okay